src/js/heuristicblocking.js

```diff
@@ -27,6 +27,7 @@ require.scopes.heuristicblocking = (function() {
 
 // make heuristic obj with utils and storage properties and put the things on it
 var tabOrigins = { }; // TODO roll into tabData?
+var tabURLs = { };
```

Comment:
Should get cleaned up when `tabOrigins` gets cleaned up. Oh wait ... Is there no cleanup? Hmm ...